src/Notification/Notification.less

```diff
@@ -22,10 +27,10 @@
 		&::before {
 			content: '';
 			display: block;
-			background-color: @moon-neutral-bg-color;
-			border: 6px solid @moon-black;
-			border-radius: 30px;
-			opacity: 0.85;
```

Comment:
Ah, I couldn't consider side effect of color alpha. I just implemented based on my experience about performance between opacity and color alpha.
I couldn't remember exact number of performance difference and don't know it is still presented, so it's better to keep your implementation for preventing side effect of color alpha.